zulipterminal/helper.py

```diff
@@ -643,10 +644,20 @@ def open_media(controller: Any, media_link: str, message_id: int) -> None:
     auth = requests.auth.HTTPBasicAuth(client.email, client.api_key)
     with requests.get(media_link, auth=auth, stream=True) as r:
         r.raise_for_status()
+        content_len = r.headers.get('content-length')
+        downloaded = 0
         with open(img_path, 'wb') as f:
             for chunk in r.iter_content(chunk_size=8192):
                 if chunk:  # Filter out keep-alive new chunks.
                     f.write(chunk)
+                    downloaded += len(chunk)
+                    if content_len:
+                        status = " Downloaded {}/{} bytes"
+                        controller.view.set_footer_text(
```

Comment:
I couldn't figure out a way to test this. I was trying to use an `assert` with `...call_count == 2`, but this particular statement doesn't execute due to mocking.

I would greatly appreciate any pointers in the review.

Reply:
I'm not sure if we want to show and update the footer as rapidly as this. Rather we could show updates every fraction of a second or thereabouts. There is a progress bar widget in urwid we could use?

This seems like it could be a separate commit to build upon the basic functionality, where we could just set 'Downloading <filename>' in the footer and clear it afterwards? (or set it to temporarily be 'downloaded'?)